cargo-dist/templates/ci/github/release.yml.j2

```diff
@@ -133,6 +133,13 @@ jobs:
         with:
           name: cargo-dist-cache
           path: ~/.cargo/bin/dist
+      {{%- if need_cargo_auditable %}}
+      - name: Install cargo-auditable
```

Comment:
Hm, I realized we're putting this in the wrong place. This is before `plan`, which plans out the job but doesn't actually perform the builds. What we actually want to do here is have `cargo auditable` installed for the local builds, which is the area starting L178 in this file.

You'll want to ensure that the github matrix entry which has the `install_dist` key gets a new `install_cargo_auditable` key, and that that gets assigned with a platform-appropriate value the same way it gets assigned for dist.